cmake/script/GenerateBuildInfo.cmake

```diff
@@ -28,9 +28,12 @@ else()
   set(WORKING_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 endif()
 
-set(GIT_TAG)
-set(GIT_COMMIT)
-if(NOT "$ENV{BITCOIN_GENBUILD_NO_GIT}" STREQUAL "1")
+set(IS_SOURCE_TARBALL FALSE)
+# git will put "set(IS_SOURCE_TARBALL TRUE)" on the next line inside archives. $Format:%nset(IS_SOURCE_TARBALL TRUE)$
```

Comment:
Maybe we should use `$Format:%(describe)%$` to actually include build info?

Reply:
This way we can put the *correct* build info in, rather than just failing to get any build info.